parquet/src/schema/printer.rs

```diff
@@ -137,6 +137,7 @@ fn print_column_chunk_metadata(
     writeln!(out, "file path: {file_path_str}");
     writeln!(out, "file offset: {}", cc_metadata.file_offset());
     writeln!(out, "num of values: {}", cc_metadata.num_values());
+    writeln!(out, "Compression: {}", cc_metadata.compression());
```

Comment:
Use lower-case to have consistent style with other info:

```suggestion
    writeln!(out, "compression: {}", cc_metadata.compression());
```

Reply:
Sorry about this, done.